### PR TITLE
Feature/jdk8

### PR DESCRIPTION
**Pull request by Dalamar42**

### CODE REVIEW

File: src/test/java/org/atlasapi/media/channel/MongoChannelStoreRetrievalTest.java [hunk 1]
```diff
@@ -167,9 +182,12 @@ public void testRetrievesChannelByGenre() {
...
     @Test
     public void testRetrievesOldEpisodes() {
-        ChannelQuery query = ChannelQuery.builder().withAdvertisedOn(dateTime).build();
+        ChannelQuery query = ChannelQuery.builder()
+                .withAdvertisedOn(dateTime.minusHours(1))
+                .build();
+
         Iterable<Channel> channels = store.allChannels(query);
-        assertThat(Iterables.size(channels), is(2));
+        assertThat(Iterables.size(channels), is(1));
```
Comment (atish-metabroadcast) [inline src/test/java/org/atlasapi/media/channel/MongoChannelStoreRetrievalTest.java]: Why the change here?

Comment (Dalamar42) [inline src/test/java/org/atlasapi/media/channel/MongoChannelStoreRetrievalTest.java]: That's odd. I was sure I pulled master. I think someone fixes the broken test in a different way than me. I changed the requested date time to ensure there is no ambiguity with whether exact equality is retrieved or not (which is not very relevant) in this context anywa

**Comment by JamieAP** [comment]
LGTM
